FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -617,6 +617,7 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
     ExtDiag(idx)%cnvfac = cn_100
     ExtDiag(idx)%time_avg = .TRUE.
     ExtDiag(idx)%time_avg_kind = 'rad_swlw_min'
+    ExtDiag(idx)%coarse_graining_method = 'area_weighted'
```

Comment:
So `coarse_graining_method` exists somewhere in the code, it just needs to be configured here?

Reply:
Yup exactly.  I added the primary infrastructure that facilitates this a few years ago https://github.com/ai2cm/fv3gfs-fortran/pull/31.

Reply:
Wow, cool!